modules/ROOT/pages/migrate-client-apps.adoc

```diff
@@ -0,0 +1,149 @@
+
+* <<migrate_with_internal,Migrate client applications for APIs with Anypoint Platform client providers>>
+* <<migrate_with_external,Migrate client applications for APIs with external client Providers>>
+
+== Prerequisites
+
+Before migrating the client application, ensure that you:
+
+* Obtain the required token and credentials:
+** Client ID and client secret for the client application to migrate
+** Anypoint Platform https://anypoint.mulesoft.com/exchange/portals/anypoint-platform/f1e97bc6-315a-4490-82a7-23abe036327a.anypoint-platform/access-management-api/minor/1.0/pages/Authentication/[token]
+* Review the <<considerations,scope and limitations>> to understand the limitations and scope of the migration task
+
+[[considerations]]
+== Scope and Limitations
+
+Before you migrate your client applications to Anypoint Platform, review the following scope and limitations: 
+
+* If the organization of the API you consume has registered an external client provider, for example, PingFederate or Okta, review the section: <<external_client,Organizations with Registered External Client Providers>>.
+* Do not attempt to update the client ID and client secret of an application.
++
+This feature is not supported.
+* Do not use the same user to create more than 500 applications.
+* Do not create applications with duplicate names.
++
+Even if the identity provider (IdP) stores the duplicate applications with different client ID and client secret credentials for the applications, issues can occur.
+* Do not attempt to refresh the client ID.
++
+Refreshing the client ID is not possible unless you delete and re-register the client application.
+* If the client credentials you provide match with the credentials of an existing organization ID on the client provider, the application gets associated with that client.
++
+Conversely, If the client credentials that you provide do not match, the request does not get associated with the client and fails.
+
+[[migrate_with_internal]]
+== Migrate Client Applications for APIs with Anypoint Platform Client Providers
+
+To migrate your client application:
+
+. Open a text editor and paste the following commands:
++
+[source,text,linenums]
+----
+curl --location --request POST 'https://anypoint.mulesoft.com/exchange/api/v1/organizations/:orgId/applications' \
+--header 'Authorization: Bearer <omitted>' \
+--header 'Content-Type: application/json' \
+--data-raw '{
+    "clientId": ":myClientId",
+    "clientSecret": ":myClientSecret",
+    "grantTypes": [],
+    "name": "My Application Name",
+    "redirectUri": []
+}'
+----
+. In the command snippet, replace the `orgId` parameter with your organization Id and the content of the body with the appropriate application <<properties,properties>>. 
```

Comment:
See above suggestion.
Otherwise:
```suggestion
. Replace the `orgId` parameter with the ID of your master organization and the content of the body with the appropriate application <<properties,properties>>. 
```

Reply:
Done! Thanks!